integration-tests/cobertura/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN go install -v && \
     apt-get update && \
     apt-get install -y maven
 
-RUN git clone https://github.com/ale7714/mavanagaiata.git
+RUN git clone https://github.com/codeclimate-testing/mavanagaiata.git
```

Comment:
thoughts on renaming it something like `test-reporter-cobertura-example`?

Reply:
@maxjacobson done :)